main.go

```diff
@@ -32,18 +33,20 @@ type config struct {
 }
 
 const (
-	maxCommitBodyLenght = 72
+	trimEnding = "..."
 )
 
-func printLogAndExportEnv(gitCmd git.Git, format, env string, trim bool) error {
+var maxCommitMessageLength int
+
+func printLogAndExportEnv(gitCmd git.Git, format, env string, trimmableVariable bool) error {
 	l, err := output(gitCmd.Log(format))
 	if err != nil {
 		return err
 	}
 
-	if trim && len(l) > maxCommitBodyLenght {
-		trimmedValue := l[:maxCommitBodyLenght]
-		log.Printf("Value %s\n trimmed to =>\n%s", l, trimmedValue)
+	if trimmableVariable && len(l) > maxCommitMessageLength {
+		trimmedValue := l[:maxCommitMessageLength] + trimEnding
+		log.Printf("Value %s trimmed", env)
```

Comment:
pls explain why we trim the value, without that this log does not really make sense

Reply:
Added details.